app/controllers/stories_controller.rb

```diff
@@ -226,7 +227,7 @@ def handle_article_show
 
   def retrieve_curated_list_info
     user_lists = CuratedList.where(user_id: session_current_user_id)
-    @curated_lists = CuratedListSerializer.new(user_lists).to_json
+    @curated_lists = CuratedListSerializer.new(user_lists).light_to_json
```

Comment:
This is an alternative method on the serializer which returns only an array of the IDs of the associated articles for each curated list rather than the full information for each associated article.